server/service.go

```diff
@@ -229,23 +229,27 @@ func (m *BoostService) handleRegisterValidator(w http.ResponseWriter, req *http.
 		"ua":               ua,
 	})
 
-	relayRespCh := make(chan error, len(m.relays))
+	relayRespCh := make(chan error, len(m.pcs.GetAllRelays()))
 
-	for _, relay := range m.relays {
-		go func(relay RelayEntry) {
-			url := relay.GetURI(pathRegisterValidator)
-			log := log.WithField("url", url)
+	for _, registration := range payload {
```

Comment:
this loop logic seems wrong. 

now its doing: for each payload, send the full payload to each relay that proposer enabled.

let's say 10 registrations and using 2 relays -- this will send all the registrations 10 times to both relays 🤔